<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Saab</class>
 <widget class="QMainWindow" name="Saab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="SAAB" stdset="0">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="grid_layout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="line_edit_query">
      <property name="text">
       <string>SELECT name FROM sqlite_master WHERE type='table';</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTableView" name="table_view"/>
    </item>
    <item row="2" column="1">
     <widget class="QCommandLinkButton" name="button_query">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Execute Query</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
